web/app/js/components/Octopus.jsx

```diff
@@ -92,6 +103,16 @@ class Octopus extends React.Component {
     return display;
   }
 
+  refCallback = (element, index, isOutbound) => {
```

Comment:
could this be named something more descriptive, for readability? Am I correct in thinking that `refCallback` adds the element to the appropriate array (`downstreamRefs` or `upstreamRefs`) so that the spacing will be correct?